XenModel/Messages.resx

```diff
@@ -972,6 +972,18 @@
   <data name="ACTION_UPLOAD_SERVER_STATUS_REPORT_PROGRESS" xml:space="preserve">
     <value>Uploading report to [Citrix] Insight Services</value>
   </data>
+  <data name="ACTION_USB_PASSTHROUGH_DISABLED" xml:space="preserve">
+    <value>USB device detached.</value>

```

Comment:
The action's title/description should reflect the UI, so if the button says "enable passthrough", then the action should also say "enabling passthrough".
ACTION_USB_PASSTHROUGH_DISABLED: USB passthrough disabled
ACTION_USB_PASSTHROUGH_DISABLING: Disabling USB passthrough for {0}
ACTION_USB_PASSTHROUGH_ENABLED: USB passthrough enabled
ACTION_USB_PASSTHROUGH_ENABLING: Enabling USB passthrough for {0}

Reply:
Thanks.